index.html

```diff
@@ -27,6 +27,157 @@ <h2>I’m a software developer</h2>
         <li><a href="#"><img src="angellist.png" alt="angellist"></a></li>    
     </ul>  
     </section>
+
+    <header class="title">
+        <div><a href="title"><img src="Disabled.png" alt="scroll down button"></a></div>
+        <div><h1>My recent Work</h1></div>
+        <div><img src="indicator.png" alt="indicator"></div>
+    </header>
+    <section class="descript">
+        <div class="planholder">
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
```

Comment:
- [ ] [Optional] This is optional, you can consider removing ```button``` for the tags, it doesn't serve any purpose

Reply:
I´ve changed the way you suggested and this was the result. I think it looked better before. Or you think I should apply a border to it instead of buttons?
![image](https://user-images.githubusercontent.com/103516637/171198149-aab5e945-8d57-42a3-9136-530ea76004b8.png)